internal/network_acl_test.go

```
@@ -0,0 +1,47 @@
+}
+
+func TestPrefixNetworkAclEvidenceTitles(t *testing.T) {
+	evidences := []*proto.Evidence{
+		{Title: "Network ACL should set required tags"},
+		{Title: ""},
+		nil,
+	}
+
+	PrefixEvidenceTitles(evidences, "acl-123")
+
+	if got := evidences[0].Title; got != "acl-123 | Network ACL should set required tags" {
```

golangci-lint (2.12.2)

[error] avoid direct access to proto field evidences[0].Title, use evidences[0].GetTitle() instead

(protogetter)